lib/vicadmin/validate.go

```diff
@@ -209,14 +204,33 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 		v.DockerPort = fmt.Sprintf("%d", opts.DefaultTLSHTTPPort)
 	}
 
-	err = v.QueryDatastore(ctx, vch, sess)
+	err := v.QueryDatastore(ctx, vch, sess)
 	if err != nil {
 		log.Errorf("Had a problem querying the datastores: %s", err.Error())
 	}
 	v.QueryVCHStatus(vch, sess)
 	return v
 }
 
+func GetVCHName(ctx context.Context, sess *session.Session) string {
```

Comment:
Nit-pick: Go's canonical way of naming "getters" recommends excluding "Get" from the function name - so this one would be `VCHName`.

https://golang.org/doc/effective_go.html#Getters

This isn't a getter per se, but anyway.